kubernetes/linux/main.sh

```diff
@@ -654,13 +673,17 @@ fi
 #If config parsing was successful, a copy of the conf file with replaced custom settings file is created
 if [ ! -e "/etc/config/kube.conf" ]; then
       if [ "${CONTAINER_TYPE}" == "PrometheusSidecar" ] && [ -e "/opt/telegraf-test-prom-side-car.conf" ]; then
-            echo "****************Start Telegraf in Test Mode**************************"
-            /opt/telegraf --config /opt/telegraf-test-prom-side-car.conf --input-filter file -test
-            if [ $? -eq 0 ]; then
-                  mv "/opt/telegraf-test-prom-side-car.conf" "/etc/opt/microsoft/docker-cimprov/telegraf-prom-side-car.conf"
-                  echo "Moving test conf file to telegraf side-car conf since test run succeeded"
+            if [ "${MUTE_PROM_SIDECAR}" != "true" ]; then
+                  echo "****************Start Telegraf in Test Mode**************************"
+                  /opt/telegraf --config /opt/telegraf-test-prom-side-car.conf --input-filter file -test
+                  if [ $? -eq 0 ]; then
+                        mv "/opt/telegraf-test-prom-side-car.conf" "/etc/opt/microsoft/docker-cimprov/telegraf-prom-side-car.conf"
+                        echo "Moving test conf file to telegraf side-car conf since test run succeeded"
+                  fi
+                  echo "****************End Telegraf Run in Test Mode**************************"
+            else
+                  echo "****************Skipping Telegraf Run in Test Mode**************************"
```

Comment:
I think updating to this "Skipping Telegraf Run in Test Mode since MUTE_PROM_SIDECAR is true" will be helpful for the troubleshooting purpose of the customer incident.

Reply:
Added the same